Yosemite/Yosemite/Stores/ProductCategoryStore.swift

```diff
@@ -109,3 +141,9 @@ private extension ProductCategoryStore {
         }
     }
 }
+
+// MARK: - Default
+//
+private extension Store.Default {
+    static let maxPageSize = 100 // Max number allwed by the API to maximize our changces on getting all items in one request.
+}
```

Comment:
🛑 I'm not sure about this. I think it's bad practice to extend a superclass' constant enum. It's a bit tricky to navigate the code using _Jump to Definition_. That will lead to this block and it'll take you a bit of time to realize that it's an `extension`. 

I think the alternative would be more obvious:

```suggestion
private extension ProductCategoryStore {
    enum Constants {
        /// Max number allwed by the API to maximize our changces on getting all items
        /// in one request.
        static let defaultMaxPageSize = 100
    }
}
```

What do you think?

Reply:
Yeah, that looks more straight forward. https://github.com/woocommerce/woocommerce-ios/pull/2130/commits/bf0ce08614d7713e14d70007c359e5b32fe49c39